colour/io/tm2714.py

```diff
@@ -601,6 +601,34 @@ def comments(self, value: Optional[str]):
 
         self._comments = value
 
+    def __eq__(self, other):
```

Comment:
We will need the typing annotations and docstrings here please, a good example: https://github.com/colour-science/colour/blob/develop/colour/continuous/signal.py#L835